src/pages/Embeds/Plio.vue

```diff
@@ -602,20 +605,21 @@ export default {
         this.isAspectRatioChecked = true;
       }
     },
-    ifAllQuestionsAreSurvey() {
+    areAllQuestionsSurvey() {
       let count = 0;
-      for (let itemIndex of this.itemDetails)
-        if (this.itemDetails[itemIndex].survey == true) count += 1;
-      return count;
+      for (let itemDetail of this.itemDetails) {
+        if (itemDetail.survey) count += 1;
+      }
+      return count == this.itemDetails.length;
     },
     /**
      * Show the scorecard on top of the player
      */
     popupScorecard() {
-      if (this.numSkipped == this.ifAllQuestionsAreSurvey) {
-        this.isScorecardShown = false;
-        this.numSkipped = 0;
+      if (this.areAllQuestionsSurvey()) {
+        return;
       }
```

Comment:
braces not needed + assuming you are changing it to a computed property:
```suggestion
      if (this.areAllQuestionsSurvey) return;
```